greenplumpython/expr.py

```diff
@@ -45,6 +46,14 @@ def __ne__(self, other):
         return BinaryExpr("!=", self, other)
 
     def __str__(self) -> str:
+        return self._mystr() + (" AS " + self._as_name if self._as_name is not None else "")
+
+    def rename(self, new_name: str) -> "Expr":
+        new_expr = copy.copy(self)  # Shallow copy
+        new_expr._as_name = new_name
+        return new_expr
+
+    def _mystr(self) -> str:
```

Comment:
`_mystr` is really not a good name.
What about `_sql_str`? Or we can change it later when a better name comes to mind.